framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -130,6 +130,39 @@ int cfg_get_certification_mode()
 int cfg_get_load_steer_on_vaps(int num_of_interfaces,
                                char load_steer_on_vaps[BPL_LOAD_STEER_ON_VAPS_LEN])
 {
+    if (num_of_interfaces < 1) {
+        MAPF_ERR("cfg_get_load_steer_on_vaps: invalid input: max num_of_interfaces value < 1");
```

Comment:
```suggestion
        MAPF_ERR("invalid input: max num_of_interfaces value < 1");
```
Same for all other places. Reason - we want to get functions easilly so adding the function name to each print will make grep produce too much output.

Reply:
was following the syntax used by all current implementations in the bpl_cfg - both for UCI and Linux implementations.
We can add a cleanup task if this is the new approach

Reply:
fixed my changes as you suggested